bittensor/wallet.py

```diff
@@ -192,14 +198,18 @@ def __init__(
         if config is None:
             config = wallet.config()
         self.config = copy.deepcopy(config)
+
+        # TODO: Do we even need the bittensor.defaults anymore?
+        #  The default values are already merged onto the config object.
```

Comment:
I don't think we do. We should only ever need the config object, the args_no_defaults object, and any defaults that are built into the lower classes like commands.